libraries/engage/core/config/getPageConfig.js

```diff
@@ -0,0 +1,19 @@
+import { getThemeConfig } from './getThemeConfig';
+
+/**
+ * This function reads the config of a page, given by its pattern. If no page is found, the result
+ * is an empty object.
+ *
+ * @param {string} pagePattern The page to look for in the theme config
+ * @returns {Object}
+ */
+export function getPageConfig(pagePattern) {
```

Comment:
The ticket is about separating out any react or router logic for flexibility. This approach keeps komplexity low by doing what it's supposed to and not doing any decisions (or assumptions about how it will be used).

Reply:
Was a misunderstanding. Cleared up. Will refactor code to accept feedback.